src/core/handlers/base.ts

```diff
@@ -234,7 +341,14 @@ export const baseHandler: LanguageHandler = {
     cells: QuartoMdCell[],
   ): MappedString[] {
     this.documentStart(handlerContext);
-    const result = cells.map((cell) => this.cell(handlerContext, cell));
+    const result = cells.map((cell) => {
+      return this.cell(
+        handlerContext,
+        cell,
+        // FIXME this should get the project+document options as well.
+        mergeConfigs(this.defaultOptions ?? {}, cell.options ?? {}),
```

Comment:
Do you want to fix this before we merge?

Reply:
Yes, sounds good. (In full disclosure, the same problem currently exists for cell handlers on `main`)